amivapi/events/model.py

```diff
@@ -436,6 +436,12 @@
                 'json_schema': True,
                 'only_if_not_null': 'spots',
             },
+            'moderator_signup_text': {
+                "description": "Custom Text from the Moderator that will be sent with the Signup-Email",
```

Comment:
The description should also be updated accordingly. You should also use more precise words.

For example:

```suggestion
                "description": "Additional information from the organizers that will be sent with the signup-accepted email.",
```